packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -19,18 +14,10 @@ import wrap from './transforms/wrap';
  */
 const transformStyles = ( styles, wrapperClassName = '' ) => {
 	return Object.values( styles ?? [] ).map( ( { css, baseURL } ) => {
-		const transforms = [];
-		if ( wrapperClassName ) {
-			transforms.push( wrap( wrapperClassName ) );
-		}
-		if ( baseURL ) {
-			transforms.push( urlRewrite( baseURL ) );
-		}
-		if ( transforms.length ) {
-			return traverse( css, compose( transforms ) );
-		}
-
-		return css;
+		return postcss( [
+			wrap( { scopeTo: wrapperClassName } ),
+			rebaseUrl( { rootUrl: baseURL } ),
```

Comment:
Nice! It's very smart to use existing transformers instead of building in-house ones.

Reply:
Though the `rebaseUrl` had to be created by me, as there are no plugins yet that do exactly that.